test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsFileParsingTests/NamespaceItemTests.cs

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using FluentAssertions;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.Configuration.Test
+{
+    public class NamespaceItemTests
+    {
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        public void Constructor_WithInvalidNamespace_Throws(string name)
+        {
+            Assert.Throws<ArgumentException>(() => new NamespaceItem(name));
+        }
+
+        [Theory]
+        [InlineData("NuGet.Common", "NuGet.Common")]
+        [InlineData("nuget.common", "NuGet.Common")]
+        [InlineData("NuGet.*", "NuGet.*")]
+        [InlineData("NuGet.*", "nuget.*")]
+        [InlineData("*", "*")]
```

Comment:
Should these test cases be added ?

```suggestion
        [InlineData("*", "*")]
        [InlineData("*", "*.*")]
        [InlineData("NuGet.*", "*.*")]
```

Reply:
No, they are not equivalent.